src/Plugin/MultiselectFilterField/BooleanField.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_list_pages\Plugin\MultiselectFilterField;
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\facets\FacetInterface;
+use Drupal\facets\Result\Result;
+use Drupal\oe_list_pages\ListPresetFilter;
+use Drupal\oe_list_pages\MultiSelectFilterFieldPluginBase;
+
+/**
+ * Defines the boolean field type multiselect filter plugin.
+ *
+ * @MultiselectFieldFilter(
+ *   id = "boolean",
+ *   label = @Translation("Boolean field"),
+ *   field_types = {
+ *     "boolean",
+ *   },
+ *   weight = 100
+ * )
+ */
+class BooleanField extends MultiSelectFilterFieldPluginBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildDefaultValueForm(): array {
+    $facet = $this->configuration['facet'];
```

Comment:
Don't we expect always the facet to be there in all plugins? Throw an exception in the constructor if the facet is missing and then we can rely on it always being there.